lottotron.js

```diff
@@ -3,6 +3,7 @@ const { isNumber } = require('util');
 const ERROR_MSG ={
   MAX_NUMBER_IS_NOT_NUMBER: 'The input option "maxNumber" should be a number.',
   MAX_NUMBER_LOWER_ZERO: 'The input option "maxNumber" should be greater than 0.',
+  MAX_NUMBER_IS_NOT_FINITE: 'The input option "maxNumber" should be a finite number.'
 }
 
```

Comment:
I think it will good idea to define own error class and to throw instance of them instead native one.
I'll create new issue about it.